### DISCUSSION TITLE
[firebase_analytics] <Automatic IDFA-based tracking support for iOS>

### DISCUSSION THREAD

**Discussion by MauScheff** [firebase/flutterfire#5597] [Feature request]
I'm sot sure if this is or isn't being done. If I want to track IDFA on iOS, should I install a pod as suggested here: https://developers.google.com/analytics/devguides/collection/ios/v3/optional-features

Or does this package handle that by default? I'm not sure what's the best way to accomplish this.

Thank you,
Mauricio

**Comment by Ehesp** [comment]
I believe this is automatically tracked:

![image](https://user-images.githubusercontent.com/842078/79961266-5aa91e80-847e-11ea-9669-b845992f71bf.png)

https://firebase.google.com/docs/analytics/configure-data-collection?platform=ios#disable_idfa_collection

This does need better documentation. Will leave open to track.
